media_kit/lib/src/player/native/player/real.dart

```diff
@@ -2455,10 +2456,11 @@ class NativePlayer extends PlatformPlayer {
   }
 
   /// Adds an error to the [Player.stream.error].
-  void _error(int code) {
-    if (code < 0 && !errorController.isClosed) {
+  void _logError(int code, String? extraContext) {
```

Comment:
Can we rename `extraContext` to `text`?

```suggestion
  void _logError(int code, String? text) {
```